app/Http/Controllers/TransactionController.php

```diff
@@ -2,6 +2,9 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Transaction;
+use Illuminate\Support\Facades\DB;
```

Comment:
@Dxhxm88 not sure why you added DB here, alll other changes are good

Reply:
Ops my bad, I forgot to remove it.